src/memory/attribute.rs

```diff
@@ -213,6 +222,277 @@ impl Topology {
         }
         polymorphized(self, target.into())
     }
+
+    /// Dump the values of all built-in memory attributes
+    pub(crate) fn dump_builtin_attributes(&self) -> MultiAttributeDump<'_> {
+        MultiAttributeDump::builtins(self)
+    }
+}
```

Comment:
This + rest of file is about being able to compare and visualize memory attributes as part of the new Debug and PartialEq implementations of Topology.